Core/Graphics/Include/Platform/Android/Babylon/Graphics/Platform.h

```diff
@@ -14,5 +14,7 @@ namespace Babylon::Graphics
         size_t Height{};
         // MSAA sample values can be 2, 4, 8 or 16. Any other value will disable MSAA
         uint8_t MSAASamples{};
+        // When enabled, back buffer will be premultiplied with alpha value
+        bool AlphaPremultiplied{};
```

Comment:
Should we rename this to `TransparentBackbuffer` like in bgfx?

Reply:
I don't think so. There is a strong link between alpha premultiplied back buffer and transparency. Alpha premul is a post process needed in a particular case of view transparency. On iOS/Android, transparency is with straight alpha. And on these 2 platforms, setting true to this boolean property will not make the view transparent.

Reply:
Ok, I don't quite understand this flag, but we can discuss offline.

Reply:
Can you add a comment on what platforms are currently supported and then remove them once they are all supported?

Reply:
Done.